client/src/components/EmotionForm.jsx

```diff
@@ -6,11 +6,9 @@ import EmoEntry from "./EmoEntry";
 import { toast } from "react-toastify";
 import "react-toastify/dist/ReactToastify.css";
 
-/* functional component will: facilitate the selection of emotions, tracks, and decisions,
-communicate with the server to fetch valence score,
-create entries in database. */
-
+// accept Spotify API JSON data as a prop
 const EmotionForm = ({ audioFeatures, savedTracks }) => {
+  // declare array of emotions
```

Comment:
You could create a constants utility file that declares all of these variables and exports them in case they're used in multiple places. 

(something to note for the future as you deem it useful)

Reply:
Great suggestion, once I start the next phase of this app, I will.